article/app/model/dotcomponents/LinkedData.scala

```diff
@@ -0,0 +1,366 @@
+package model.dotcomponents
```

Comment:
This is the old LinkedData file (from common) moved here as nearly all of it is only used by dotcomponents and relevant to that.

But obviously, the liveblog-specific parts are new!